docs/working-with-projects.md

```diff
@@ -0,0 +1,31 @@
+# Working with Projects
+
+**Projects** are used to group clusters, to onboard IaaS resources utilized by them and organize access control. 
+To work with clusters, you need to create a project that they’ll belong to.
+
+### Prerequisites
+- You have access to the Gardener dashboard and have permissions to create projects.
+
+### Procedure
+
+1. Log on to the Gardener Dashboard and choose **CREATE YOUR FIRST PROJECT**.
+
+    <img class="gs-image" src="images/00-create-project.png">
+    
+2. Provide a project **Name**, and optionally a **Description**, and a **Purpose**, and choose **CREATE**.
+
+    **Note:** You will **not** be able to change the project **Name** later. The rest of the details are editable.
+
+    <img class="gs-image" style="max-width:65%" src="images/01-provide-project-details.png">
+
+    The result is similar to the following:
+
+    <img class="gs-image" src="images/02-create-project-done.png">
```

Comment:
The result shows trial secrets. This is specific to how SAP did set up their landscape and automatically populates trial secret(binding)s to new projects.